src/node/ext/server.cc

```diff
@@ -242,7 +277,9 @@ NAN_METHOD(Server::TryShutdown) {
     return Nan::ThrowTypeError("tryShutdown can only be called on a Server");
   }
   Server *server = ObjectWrap::Unwrap<Server>(info.This());
+  TryShutdownOp *op = new TryShutdownOp(server, info.This());
```

Comment:
it seems like `TryShutdown` ends up doing the same thing as `ForceShutdown`, except that there's no call to `grpc_server_cancel_all_calls(this->wrapped_server);`. But `TryShutdown` ends up using the `OnComplete` op callback while `ForceShutdown` is using the tag's callback. Is it possible for `TryShutdown` to also use the tag callback?